config/image_kinds.yml

```diff
@@ -29,6 +29,36 @@ default:
       width: 216
       height: 140
       from_version: s960
+landing_page_image:
+  display_name: "Landing page image (2x pixel density) (1260 by 944)"
+  valid_width: 1260
+  valid_height: 944
+  permitted_uses:
+    - landing_page
+  versions:
+    - name: landing_page_desktop_2x
+      width: 1260
+      height: 944
+    - name: landing_page_desktop_1x
+      width: 630
+      height: 472
+      from_version: landing_page_desktop_2x
+    - name: landing_page_tablet_2x
+      width: 1416
+      height: 1062
```

Comment:
Hmmm shouldn't this be the value used for `valid_width`/`valid_height`? I understood that to be "the largest of the versions" so that it can scale down...?

Reply:
Ahh yeah - I hadn't realised the tablet image is actually bigger than the desktop image. Yes, you're right. That should be the main one, and the others should scale down from it. Good catch.